Formula/m/mdz.rb

```diff
@@ -0,0 +1,46 @@
+class Mdz < Formula
+  desc "CLI for the mdz ledger Open Source"
+  homepage "https://github.com/LerianStudio/midaz"
+  url "https://github.com/LerianStudio/midaz/archive/refs/tags/v1.31.0.tar.gz"
+  sha256 "16b0e877091c1ef6fc466e52c4281a9407081459264f3c8bb7917699687224fa"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X github.com/LerianStudio/midaz/components/mdz/pkg/environment.ClientID=\
+      9670e0ca55a29a466d31
+      -X github.com/LerianStudio/midaz/components/mdz/pkg/environment.ClientSecret=\
+      dd03f916cacf4a98c6a413d9c38ba102dce436a9
```

Comment:
what are these crypto values?

Reply:
These are just example values, the command being tested configure sets them to use cli as well